src/EPR.SubsidiaryBulkUpload.Function/BulkUploadFunction.cs

```diff
@@ -44,7 +44,7 @@ public async Task Run(
             }
         }
 
-        var userGuid = metaData.Where(pair => pair.Key.Contains("userId"))
```

Comment:
I don't think you need any of this - userRequestModel has the userId, and if it doesn't find userId or the organisationId it will return null. You could use that for logging instead.

If you still need to keep this code, it would be better to use `metaData.Where(pair => string.Compare(pair.Key, "userId", StringComparison.OrdinalIgnoreCase) == 0)`  for an exact match. Your current code will pick up something like "not a userid" and treat it as a userid.

Reply:
block removed as suggested by you.